pyzeebe/client/client.py

```diff
@@ -2,28 +2,26 @@
 
 import grpc
 
-from pyzeebe.credentials.base_credentials import BaseCredentials
 from pyzeebe.grpc_internals.zeebe_adapter import ZeebeAdapter
 
 
 class ZeebeClient(object):
     """A zeebe client that can connect to a zeebe instance and perform actions."""
 
-    def __init__(self, hostname: str = None, port: int = None, credentials: BaseCredentials = None,
-                 secure_connection: bool = False, max_connection_retries: int = 10):
+    def __init__(
+        self, grpc_channel: grpc.aio.Channel, max_connection_retries: int = 10
+    ):
```

Comment:
Started to use Black?

Reply:
Yes, I'm considering using it in pyzeebe :)